2022-07-12 12:06:17,499 [http-nio-8081-exec-4] INFO  (RpcConnection.java [createCollection]:184) - collection /db/dokumenti has been created 
2022-07-12 12:06:17,540 [http-nio-8081-exec-4] INFO  (RpcConnection.java [createCollection]:184) - collection /db/dokumenti/zahtevZaZeleniSertifikat has been created 
2022-07-12 17:46:23,435 [http-nio-8081-exec-7] INFO  (RpcConnection.java [createCollection]:184) - collection /db/vaccines has been created 
2022-07-12 17:46:23,481 [http-nio-8081-exec-5] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-07-12 17:52:56,080 [http-nio-8081-exec-1] INFO  (RpcConnection.java [doQuery]:257) - query took 17ms. 
2022-08-10 17:11:26,766 [http-nio-8081-exec-9] INFO  (RpcConnection.java [createCollection]:184) - collection /db/gradjanin has been created 
2022-08-10 17:11:26,822 [http-nio-8081-exec-9] INFO  (RpcConnection.java [createCollection]:184) - collection /db/gradjanin/xml has been created 
2022-08-10 17:11:26,870 [http-nio-8081-exec-6] INFO  (RpcConnection.java [createCollection]:184) - collection /db/gradjanin/xml/korisnik has been created 
2022-08-10 19:09:40,812 [http-nio-8081-exec-3] INFO  (RpcConnection.java [createCollection]:184) - collection /db/dokumenti/zahtevZaSaglasnost has been created 
2022-08-10 20:49:28,600 [http-nio-8081-exec-6] INFO  (RpcConnection.java [createCollection]:184) - collection /db/dokumenti/saglasnost has been created 
2022-08-13 07:12:42,727 [http-nio-8081-exec-2] INFO  (RpcConnection.java [createCollection]:184) - collection /db/dokumenti/izvestajOImunizaciji has been created 
2022-08-13 07:12:42,774 [http-nio-8081-exec-7] ERROR (XmlRpcErrorLogger.java [log]:36) - Failed to invoke method parse in class org.exist.xmlrpc.RpcConnection: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (-1,-1) : Premature end of file.
org.xml.sax.SAXException: fatal error at (-1,-1) : Premature end of file.
org.xml.sax.SAXParseException; Premature end of file. 
org.apache.xmlrpc.common.XmlRpcInvocationException: Failed to invoke method parse in class org.exist.xmlrpc.RpcConnection: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (-1,-1) : Premature end of file.
org.xml.sax.SAXException: fatal error at (-1,-1) : Premature end of file.
org.xml.sax.SAXParseException; Premature end of file.
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:129) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.execute(ReflectiveXmlRpcHandler.java:106) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServerWorker.execute(XmlRpcServerWorker.java:46) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServer.execute(XmlRpcServer.java:86) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcStreamServer.execute(XmlRpcStreamServer.java:200) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServletServer.execute(XmlRpcServletServer.java:112) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServlet.doPost(XmlRpcServlet.java:196) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.exist.xmlrpc.RpcServlet.doPost(RpcServlet.java:99) [exist.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:681) [servlet-api.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:462) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:356) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:316) [catalina.jar:8.5.73]
	at org.exist.http.urlrewrite.Forward.doRewrite(Forward.java:51) [exist-optional.jar:4.8.0]
	at org.exist.http.urlrewrite.XQueryURLRewrite.service(XQueryURLRewrite.java:206) [exist-optional.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52) [tomcat-websocket.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.logging.log4j.web.Log4jServletFilter.doFilter(Log4jServletFilter.java:71) [log4j-web-2.15.0.jar:2.15.0]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:196) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97) [catalina.jar:8.5.73]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:698) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78) [catalina.jar:8.5.73]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:364) [catalina.jar:8.5.73]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:624) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:831) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1673) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-util.jar:8.5.73]
	at java.lang.Thread.run(Thread.java:750) [?:1.8.0_342]
Caused by: org.exist.EXistException: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (-1,-1) : Premature end of file.
org.xml.sax.SAXException: fatal error at (-1,-1) : Premature end of file.
org.xml.sax.SAXParseException; Premature end of file.
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:49) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_342]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_342]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
Caused by: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (-1,-1) : Premature end of file.
	at org.exist.collections.MutableCollection.lambda$validateXMLResource$3(MutableCollection.java:1305) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResourceInternal(MutableCollection.java:1422) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResource(MutableCollection.java:1290) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$parse$42(RpcConnection.java:1381) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:47) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_342]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_342]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
Caused by: org.xml.sax.SAXException: fatal error at (-1,-1) : Premature end of file.
	at org.exist.Indexer.fatalError(Indexer.java:471) ~[exist.jar:4.8.0]
	at org.apache.xerces.util.ErrorHandlerWrapper.fatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLVersionDetector.determineDocVersion(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.exist.collections.MutableCollection.lambda$validateXMLResource$3(MutableCollection.java:1303) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResourceInternal(MutableCollection.java:1422) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResource(MutableCollection.java:1290) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$parse$42(RpcConnection.java:1381) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:47) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_342]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_342]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
Caused by: org.xml.sax.SAXParseException: Premature end of file.
	at org.apache.xerces.util.ErrorHandlerWrapper.createSAXParseException(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.util.ErrorHandlerWrapper.fatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLVersionDetector.determineDocVersion(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.exist.collections.MutableCollection.lambda$validateXMLResource$3(MutableCollection.java:1303) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResourceInternal(MutableCollection.java:1422) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResource(MutableCollection.java:1290) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$parse$42(RpcConnection.java:1381) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:47) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_342]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_342]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
2022-08-21 18:56:22,414 [http-nio-8081-exec-5] INFO  (RpcConnection.java [createCollection]:184) - collection /db/requests has been created 
2022-08-21 18:56:22,486 [http-nio-8081-exec-10] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-21 18:58:53,571 [http-nio-8081-exec-6] INFO  (RpcConnection.java [doQuery]:257) - query took 18ms. 
2022-08-21 20:55:01,622 [http-nio-8081-exec-9] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-21 20:56:04,519 [http-nio-8081-exec-4] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-21 20:56:15,830 [http-nio-8081-exec-9] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-21 20:59:39,708 [http-nio-8081-exec-5] ERROR (XmlRpcErrorLogger.java [log]:36) - Failed to invoke method retrieveFirstChunk in class org.exist.xmlrpc.RpcConnection: result set unknown or timed out: 4 
org.apache.xmlrpc.common.XmlRpcInvocationException: Failed to invoke method retrieveFirstChunk in class org.exist.xmlrpc.RpcConnection: result set unknown or timed out: 4
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:129) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.execute(ReflectiveXmlRpcHandler.java:106) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServerWorker.execute(XmlRpcServerWorker.java:46) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServer.execute(XmlRpcServer.java:86) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcStreamServer.execute(XmlRpcStreamServer.java:200) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServletServer.execute(XmlRpcServletServer.java:112) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServlet.doPost(XmlRpcServlet.java:196) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.exist.xmlrpc.RpcServlet.doPost(RpcServlet.java:99) [exist.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:681) [servlet-api.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:462) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:356) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:316) [catalina.jar:8.5.73]
	at org.exist.http.urlrewrite.Forward.doRewrite(Forward.java:51) [exist-optional.jar:4.8.0]
	at org.exist.http.urlrewrite.XQueryURLRewrite.service(XQueryURLRewrite.java:206) [exist-optional.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52) [tomcat-websocket.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.logging.log4j.web.Log4jServletFilter.doFilter(Log4jServletFilter.java:71) [log4j-web-2.15.0.jar:2.15.0]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:196) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97) [catalina.jar:8.5.73]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:698) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78) [catalina.jar:8.5.73]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:364) [catalina.jar:8.5.73]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:624) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:831) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1673) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-util.jar:8.5.73]
	at java.lang.Thread.run(Thread.java:750) [?:1.8.0_342]
Caused by: org.exist.EXistException: result set unknown or timed out: 4
	at org.exist.xmlrpc.RpcConnection.lambda$retrieveFirstChunk$72(RpcConnection.java:2275) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.retrieveFirstChunk(RpcConnection.java:2272) ~[exist.jar:4.8.0]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_342]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_342]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
2022-08-21 20:59:47,422 [http-nio-8081-exec-2] ERROR (XmlRpcErrorLogger.java [log]:36) - Failed to invoke method retrieveFirstChunk in class org.exist.xmlrpc.RpcConnection: result set unknown or timed out: 4 
org.apache.xmlrpc.common.XmlRpcInvocationException: Failed to invoke method retrieveFirstChunk in class org.exist.xmlrpc.RpcConnection: result set unknown or timed out: 4
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:129) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.execute(ReflectiveXmlRpcHandler.java:106) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServerWorker.execute(XmlRpcServerWorker.java:46) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServer.execute(XmlRpcServer.java:86) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcStreamServer.execute(XmlRpcStreamServer.java:200) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServletServer.execute(XmlRpcServletServer.java:112) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServlet.doPost(XmlRpcServlet.java:196) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.exist.xmlrpc.RpcServlet.doPost(RpcServlet.java:99) [exist.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:681) [servlet-api.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:462) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:356) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:316) [catalina.jar:8.5.73]
	at org.exist.http.urlrewrite.Forward.doRewrite(Forward.java:51) [exist-optional.jar:4.8.0]
	at org.exist.http.urlrewrite.XQueryURLRewrite.service(XQueryURLRewrite.java:206) [exist-optional.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52) [tomcat-websocket.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.logging.log4j.web.Log4jServletFilter.doFilter(Log4jServletFilter.java:71) [log4j-web-2.15.0.jar:2.15.0]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:196) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97) [catalina.jar:8.5.73]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:698) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78) [catalina.jar:8.5.73]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:364) [catalina.jar:8.5.73]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:624) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:831) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1673) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-util.jar:8.5.73]
	at java.lang.Thread.run(Thread.java:750) [?:1.8.0_342]
Caused by: org.exist.EXistException: result set unknown or timed out: 4
	at org.exist.xmlrpc.RpcConnection.lambda$retrieveFirstChunk$72(RpcConnection.java:2275) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.retrieveFirstChunk(RpcConnection.java:2272) ~[exist.jar:4.8.0]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_342]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_342]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
2022-08-21 21:02:27,909 [http-nio-8081-exec-4] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-21 21:03:24,349 [http-nio-8081-exec-4] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-21 21:03:28,272 [http-nio-8081-exec-9] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-22 16:25:07,927 [http-nio-8081-exec-2] INFO  (RpcConnection.java [doQuery]:257) - query took 2ms. 
2022-08-22 16:46:02,828 [http-nio-8081-exec-2] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-22 16:46:10,717 [http-nio-8081-exec-9] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-22 16:46:26,324 [http-nio-8081-exec-10] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-22 16:46:31,772 [http-nio-8081-exec-4] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-22 16:46:45,436 [http-nio-8081-exec-5] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-22 16:47:11,562 [http-nio-8081-exec-5] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-22 16:47:21,133 [http-nio-8081-exec-1] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-22 16:47:35,814 [http-nio-8081-exec-2] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-22 16:47:42,255 [http-nio-8081-exec-9] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-22 16:48:13,732 [http-nio-8081-exec-9] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-22 16:48:44,941 [http-nio-8081-exec-9] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-22 16:49:12,194 [http-nio-8081-exec-9] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-22 16:49:37,853 [http-nio-8081-exec-9] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-22 16:49:41,429 [http-nio-8081-exec-10] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-22 16:50:53,815 [http-nio-8081-exec-10] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-22 16:51:06,349 [http-nio-8081-exec-4] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-22 16:51:15,998 [http-nio-8081-exec-5] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-22 16:51:27,559 [http-nio-8081-exec-6] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-22 16:51:40,332 [http-nio-8081-exec-1] INFO  (RpcConnection.java [createCollection]:184) - collection /db/potvrde has been created 
2022-08-22 16:51:40,355 [http-nio-8081-exec-4] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-22 16:51:40,391 [http-nio-8081-exec-5] INFO  (RpcConnection.java [createCollection]:184) - collection /db/saglasnosti has been created 
2022-08-22 16:51:40,409 [http-nio-8081-exec-7] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-22 16:51:40,618 [http-nio-8081-exec-9] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-22 16:51:40,674 [http-nio-8081-exec-3] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-22 16:51:45,412 [http-nio-8081-exec-10] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-22 16:51:49,465 [http-nio-8081-exec-2] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-22 16:51:49,480 [http-nio-8081-exec-8] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-22 21:56:30,370 [http-nio-8081-exec-1] INFO  (RpcConnection.java [doQuery]:257) - query took 2ms. 
2022-08-22 21:56:36,601 [http-nio-8081-exec-10] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-22 21:56:36,623 [http-nio-8081-exec-4] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-22 21:57:32,507 [http-nio-8081-exec-6] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-22 21:57:32,531 [http-nio-8081-exec-1] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-22 22:43:21,617 [http-nio-8081-exec-10] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-22 22:43:26,951 [http-nio-8081-exec-4] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 01:59:55,122 [http-nio-8081-exec-8] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-23 02:00:29,373 [http-nio-8081-exec-2] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 02:00:29,400 [http-nio-8081-exec-9] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 03:30:01,135 [http-nio-8081-exec-4] ERROR (XmlRpcErrorLogger.java [log]:36) - Failed to invoke method parse in class org.exist.xmlrpc.RpcConnection: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXException: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXParseException; lineNumber: 67; columnNumber: 28; The entity "nbsp" was referenced, but not declared. 
org.apache.xmlrpc.common.XmlRpcInvocationException: Failed to invoke method parse in class org.exist.xmlrpc.RpcConnection: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXException: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXParseException; lineNumber: 67; columnNumber: 28; The entity "nbsp" was referenced, but not declared.
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:129) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.execute(ReflectiveXmlRpcHandler.java:106) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServerWorker.execute(XmlRpcServerWorker.java:46) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServer.execute(XmlRpcServer.java:86) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcStreamServer.execute(XmlRpcStreamServer.java:200) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServletServer.execute(XmlRpcServletServer.java:112) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServlet.doPost(XmlRpcServlet.java:196) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.exist.xmlrpc.RpcServlet.doPost(RpcServlet.java:99) [exist.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:681) [servlet-api.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:462) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:356) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:316) [catalina.jar:8.5.73]
	at org.exist.http.urlrewrite.Forward.doRewrite(Forward.java:51) [exist-optional.jar:4.8.0]
	at org.exist.http.urlrewrite.XQueryURLRewrite.service(XQueryURLRewrite.java:206) [exist-optional.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52) [tomcat-websocket.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.logging.log4j.web.Log4jServletFilter.doFilter(Log4jServletFilter.java:71) [log4j-web-2.15.0.jar:2.15.0]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:196) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97) [catalina.jar:8.5.73]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:698) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78) [catalina.jar:8.5.73]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:364) [catalina.jar:8.5.73]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:624) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:831) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1673) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-util.jar:8.5.73]
	at java.lang.Thread.run(Thread.java:750) [?:1.8.0_342]
Caused by: org.exist.EXistException: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXException: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXParseException; lineNumber: 67; columnNumber: 28; The entity "nbsp" was referenced, but not declared.
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:49) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_342]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_342]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
Caused by: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
	at org.exist.collections.MutableCollection.lambda$validateXMLResource$3(MutableCollection.java:1305) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResourceInternal(MutableCollection.java:1422) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResource(MutableCollection.java:1290) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$parse$42(RpcConnection.java:1381) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:47) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_342]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_342]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
Caused by: org.xml.sax.SAXException: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
	at org.exist.Indexer.fatalError(Indexer.java:471) ~[exist.jar:4.8.0]
	at org.apache.xerces.util.ErrorHandlerWrapper.fatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLScanner.reportFatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.exist.collections.MutableCollection.lambda$validateXMLResource$3(MutableCollection.java:1303) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResourceInternal(MutableCollection.java:1422) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResource(MutableCollection.java:1290) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$parse$42(RpcConnection.java:1381) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:47) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_342]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_342]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
Caused by: org.xml.sax.SAXParseException: The entity "nbsp" was referenced, but not declared.
	at org.apache.xerces.util.ErrorHandlerWrapper.createSAXParseException(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.util.ErrorHandlerWrapper.fatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLScanner.reportFatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.exist.collections.MutableCollection.lambda$validateXMLResource$3(MutableCollection.java:1303) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResourceInternal(MutableCollection.java:1422) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResource(MutableCollection.java:1290) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$parse$42(RpcConnection.java:1381) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:47) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_342]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_342]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
2022-08-23 03:30:31,554 [http-nio-8081-exec-5] ERROR (XmlRpcErrorLogger.java [log]:36) - Failed to invoke method parse in class org.exist.xmlrpc.RpcConnection: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXException: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXParseException; lineNumber: 67; columnNumber: 28; The entity "nbsp" was referenced, but not declared. 
org.apache.xmlrpc.common.XmlRpcInvocationException: Failed to invoke method parse in class org.exist.xmlrpc.RpcConnection: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXException: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXParseException; lineNumber: 67; columnNumber: 28; The entity "nbsp" was referenced, but not declared.
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:129) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.execute(ReflectiveXmlRpcHandler.java:106) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServerWorker.execute(XmlRpcServerWorker.java:46) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServer.execute(XmlRpcServer.java:86) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcStreamServer.execute(XmlRpcStreamServer.java:200) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServletServer.execute(XmlRpcServletServer.java:112) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServlet.doPost(XmlRpcServlet.java:196) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.exist.xmlrpc.RpcServlet.doPost(RpcServlet.java:99) [exist.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:681) [servlet-api.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:462) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:356) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:316) [catalina.jar:8.5.73]
	at org.exist.http.urlrewrite.Forward.doRewrite(Forward.java:51) [exist-optional.jar:4.8.0]
	at org.exist.http.urlrewrite.XQueryURLRewrite.service(XQueryURLRewrite.java:206) [exist-optional.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52) [tomcat-websocket.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.logging.log4j.web.Log4jServletFilter.doFilter(Log4jServletFilter.java:71) [log4j-web-2.15.0.jar:2.15.0]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:196) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97) [catalina.jar:8.5.73]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:698) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78) [catalina.jar:8.5.73]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:364) [catalina.jar:8.5.73]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:624) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:831) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1673) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-util.jar:8.5.73]
	at java.lang.Thread.run(Thread.java:750) [?:1.8.0_342]
Caused by: org.exist.EXistException: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXException: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXParseException; lineNumber: 67; columnNumber: 28; The entity "nbsp" was referenced, but not declared.
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:49) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.GeneratedMethodAccessor53.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
Caused by: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
	at org.exist.collections.MutableCollection.lambda$validateXMLResource$3(MutableCollection.java:1305) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResourceInternal(MutableCollection.java:1422) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResource(MutableCollection.java:1290) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$parse$42(RpcConnection.java:1381) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:47) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.GeneratedMethodAccessor53.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
Caused by: org.xml.sax.SAXException: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
	at org.exist.Indexer.fatalError(Indexer.java:471) ~[exist.jar:4.8.0]
	at org.apache.xerces.util.ErrorHandlerWrapper.fatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLScanner.reportFatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.exist.collections.MutableCollection.lambda$validateXMLResource$3(MutableCollection.java:1303) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResourceInternal(MutableCollection.java:1422) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResource(MutableCollection.java:1290) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$parse$42(RpcConnection.java:1381) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:47) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.GeneratedMethodAccessor53.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
Caused by: org.xml.sax.SAXParseException: The entity "nbsp" was referenced, but not declared.
	at org.apache.xerces.util.ErrorHandlerWrapper.createSAXParseException(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.util.ErrorHandlerWrapper.fatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLScanner.reportFatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.exist.collections.MutableCollection.lambda$validateXMLResource$3(MutableCollection.java:1303) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResourceInternal(MutableCollection.java:1422) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResource(MutableCollection.java:1290) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$parse$42(RpcConnection.java:1381) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:47) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.GeneratedMethodAccessor53.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
2022-08-23 03:37:56,978 [http-nio-8081-exec-1] ERROR (XmlRpcErrorLogger.java [log]:36) - Failed to invoke method parse in class org.exist.xmlrpc.RpcConnection: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,25) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXException: fatal error at (67,25) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXParseException; lineNumber: 67; columnNumber: 25; The entity "nbsp" was referenced, but not declared. 
org.apache.xmlrpc.common.XmlRpcInvocationException: Failed to invoke method parse in class org.exist.xmlrpc.RpcConnection: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,25) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXException: fatal error at (67,25) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXParseException; lineNumber: 67; columnNumber: 25; The entity "nbsp" was referenced, but not declared.
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:129) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.execute(ReflectiveXmlRpcHandler.java:106) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServerWorker.execute(XmlRpcServerWorker.java:46) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServer.execute(XmlRpcServer.java:86) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcStreamServer.execute(XmlRpcStreamServer.java:200) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServletServer.execute(XmlRpcServletServer.java:112) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServlet.doPost(XmlRpcServlet.java:196) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.exist.xmlrpc.RpcServlet.doPost(RpcServlet.java:99) [exist.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:681) [servlet-api.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:462) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:356) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:316) [catalina.jar:8.5.73]
	at org.exist.http.urlrewrite.Forward.doRewrite(Forward.java:51) [exist-optional.jar:4.8.0]
	at org.exist.http.urlrewrite.XQueryURLRewrite.service(XQueryURLRewrite.java:206) [exist-optional.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52) [tomcat-websocket.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.logging.log4j.web.Log4jServletFilter.doFilter(Log4jServletFilter.java:71) [log4j-web-2.15.0.jar:2.15.0]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:196) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97) [catalina.jar:8.5.73]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:698) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78) [catalina.jar:8.5.73]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:364) [catalina.jar:8.5.73]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:624) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:831) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1673) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-util.jar:8.5.73]
	at java.lang.Thread.run(Thread.java:750) [?:1.8.0_342]
Caused by: org.exist.EXistException: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,25) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXException: fatal error at (67,25) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXParseException; lineNumber: 67; columnNumber: 25; The entity "nbsp" was referenced, but not declared.
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:49) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.GeneratedMethodAccessor53.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
Caused by: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,25) : The entity "nbsp" was referenced, but not declared.
	at org.exist.collections.MutableCollection.lambda$validateXMLResource$3(MutableCollection.java:1305) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResourceInternal(MutableCollection.java:1422) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResource(MutableCollection.java:1290) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$parse$42(RpcConnection.java:1381) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:47) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.GeneratedMethodAccessor53.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
Caused by: org.xml.sax.SAXException: fatal error at (67,25) : The entity "nbsp" was referenced, but not declared.
	at org.exist.Indexer.fatalError(Indexer.java:471) ~[exist.jar:4.8.0]
	at org.apache.xerces.util.ErrorHandlerWrapper.fatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLScanner.reportFatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.exist.collections.MutableCollection.lambda$validateXMLResource$3(MutableCollection.java:1303) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResourceInternal(MutableCollection.java:1422) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResource(MutableCollection.java:1290) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$parse$42(RpcConnection.java:1381) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:47) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.GeneratedMethodAccessor53.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
Caused by: org.xml.sax.SAXParseException: The entity "nbsp" was referenced, but not declared.
	at org.apache.xerces.util.ErrorHandlerWrapper.createSAXParseException(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.util.ErrorHandlerWrapper.fatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLScanner.reportFatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.exist.collections.MutableCollection.lambda$validateXMLResource$3(MutableCollection.java:1303) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResourceInternal(MutableCollection.java:1422) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResource(MutableCollection.java:1290) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$parse$42(RpcConnection.java:1381) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:47) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.GeneratedMethodAccessor53.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
2022-08-23 03:42:58,035 [http-nio-8081-exec-8] ERROR (XmlRpcErrorLogger.java [log]:36) - Failed to invoke method parse in class org.exist.xmlrpc.RpcConnection: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,25) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXException: fatal error at (67,25) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXParseException; lineNumber: 67; columnNumber: 25; The entity "nbsp" was referenced, but not declared. 
org.apache.xmlrpc.common.XmlRpcInvocationException: Failed to invoke method parse in class org.exist.xmlrpc.RpcConnection: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,25) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXException: fatal error at (67,25) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXParseException; lineNumber: 67; columnNumber: 25; The entity "nbsp" was referenced, but not declared.
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:129) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.execute(ReflectiveXmlRpcHandler.java:106) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServerWorker.execute(XmlRpcServerWorker.java:46) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServer.execute(XmlRpcServer.java:86) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcStreamServer.execute(XmlRpcStreamServer.java:200) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServletServer.execute(XmlRpcServletServer.java:112) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServlet.doPost(XmlRpcServlet.java:196) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.exist.xmlrpc.RpcServlet.doPost(RpcServlet.java:99) [exist.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:681) [servlet-api.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:462) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:356) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:316) [catalina.jar:8.5.73]
	at org.exist.http.urlrewrite.Forward.doRewrite(Forward.java:51) [exist-optional.jar:4.8.0]
	at org.exist.http.urlrewrite.XQueryURLRewrite.service(XQueryURLRewrite.java:206) [exist-optional.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52) [tomcat-websocket.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.logging.log4j.web.Log4jServletFilter.doFilter(Log4jServletFilter.java:71) [log4j-web-2.15.0.jar:2.15.0]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:196) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97) [catalina.jar:8.5.73]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:698) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78) [catalina.jar:8.5.73]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:364) [catalina.jar:8.5.73]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:624) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:831) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1673) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-util.jar:8.5.73]
	at java.lang.Thread.run(Thread.java:750) [?:1.8.0_342]
Caused by: org.exist.EXistException: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,25) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXException: fatal error at (67,25) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXParseException; lineNumber: 67; columnNumber: 25; The entity "nbsp" was referenced, but not declared.
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:49) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.GeneratedMethodAccessor53.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
Caused by: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,25) : The entity "nbsp" was referenced, but not declared.
	at org.exist.collections.MutableCollection.lambda$validateXMLResource$3(MutableCollection.java:1305) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResourceInternal(MutableCollection.java:1422) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResource(MutableCollection.java:1290) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$parse$42(RpcConnection.java:1381) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:47) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.GeneratedMethodAccessor53.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
Caused by: org.xml.sax.SAXException: fatal error at (67,25) : The entity "nbsp" was referenced, but not declared.
	at org.exist.Indexer.fatalError(Indexer.java:471) ~[exist.jar:4.8.0]
	at org.apache.xerces.util.ErrorHandlerWrapper.fatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLScanner.reportFatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.exist.collections.MutableCollection.lambda$validateXMLResource$3(MutableCollection.java:1303) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResourceInternal(MutableCollection.java:1422) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResource(MutableCollection.java:1290) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$parse$42(RpcConnection.java:1381) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:47) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.GeneratedMethodAccessor53.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
Caused by: org.xml.sax.SAXParseException: The entity "nbsp" was referenced, but not declared.
	at org.apache.xerces.util.ErrorHandlerWrapper.createSAXParseException(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.util.ErrorHandlerWrapper.fatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLScanner.reportFatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.exist.collections.MutableCollection.lambda$validateXMLResource$3(MutableCollection.java:1303) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResourceInternal(MutableCollection.java:1422) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResource(MutableCollection.java:1290) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$parse$42(RpcConnection.java:1381) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:47) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.GeneratedMethodAccessor53.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
2022-08-23 03:51:55,008 [http-nio-8081-exec-10] ERROR (XmlRpcErrorLogger.java [log]:36) - Failed to invoke method parse in class org.exist.xmlrpc.RpcConnection: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXException: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXParseException; lineNumber: 67; columnNumber: 28; The entity "nbsp" was referenced, but not declared. 
org.apache.xmlrpc.common.XmlRpcInvocationException: Failed to invoke method parse in class org.exist.xmlrpc.RpcConnection: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXException: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXParseException; lineNumber: 67; columnNumber: 28; The entity "nbsp" was referenced, but not declared.
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:129) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.execute(ReflectiveXmlRpcHandler.java:106) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServerWorker.execute(XmlRpcServerWorker.java:46) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServer.execute(XmlRpcServer.java:86) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcStreamServer.execute(XmlRpcStreamServer.java:200) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServletServer.execute(XmlRpcServletServer.java:112) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServlet.doPost(XmlRpcServlet.java:196) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.exist.xmlrpc.RpcServlet.doPost(RpcServlet.java:99) [exist.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:681) [servlet-api.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:462) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:356) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:316) [catalina.jar:8.5.73]
	at org.exist.http.urlrewrite.Forward.doRewrite(Forward.java:51) [exist-optional.jar:4.8.0]
	at org.exist.http.urlrewrite.XQueryURLRewrite.service(XQueryURLRewrite.java:206) [exist-optional.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52) [tomcat-websocket.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.logging.log4j.web.Log4jServletFilter.doFilter(Log4jServletFilter.java:71) [log4j-web-2.15.0.jar:2.15.0]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:196) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97) [catalina.jar:8.5.73]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:698) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78) [catalina.jar:8.5.73]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:364) [catalina.jar:8.5.73]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:624) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:831) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1673) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-util.jar:8.5.73]
	at java.lang.Thread.run(Thread.java:750) [?:1.8.0_342]
Caused by: org.exist.EXistException: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXException: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
org.xml.sax.SAXParseException; lineNumber: 67; columnNumber: 28; The entity "nbsp" was referenced, but not declared.
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:49) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.GeneratedMethodAccessor53.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
Caused by: org.xml.sax.SAXException: The XML parser reported a problem: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
	at org.exist.collections.MutableCollection.lambda$validateXMLResource$3(MutableCollection.java:1305) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResourceInternal(MutableCollection.java:1422) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResource(MutableCollection.java:1290) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$parse$42(RpcConnection.java:1381) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:47) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.GeneratedMethodAccessor53.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
Caused by: org.xml.sax.SAXException: fatal error at (67,28) : The entity "nbsp" was referenced, but not declared.
	at org.exist.Indexer.fatalError(Indexer.java:471) ~[exist.jar:4.8.0]
	at org.apache.xerces.util.ErrorHandlerWrapper.fatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLScanner.reportFatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.exist.collections.MutableCollection.lambda$validateXMLResource$3(MutableCollection.java:1303) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResourceInternal(MutableCollection.java:1422) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResource(MutableCollection.java:1290) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$parse$42(RpcConnection.java:1381) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:47) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.GeneratedMethodAccessor53.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
Caused by: org.xml.sax.SAXParseException: The entity "nbsp" was referenced, but not declared.
	at org.apache.xerces.util.ErrorHandlerWrapper.createSAXParseException(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.util.ErrorHandlerWrapper.fatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLScanner.reportFatalError(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source) ~[xercesImpl-2.11.0.jar:?]
	at org.exist.collections.MutableCollection.lambda$validateXMLResource$3(MutableCollection.java:1303) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResourceInternal(MutableCollection.java:1422) ~[exist.jar:4.8.0]
	at org.exist.collections.MutableCollection.validateXMLResource(MutableCollection.java:1290) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$parse$42(RpcConnection.java:1381) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcCollectionFunction.apply(XmlRpcCollectionFunction.java:47) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$withCollection$129(RpcConnection.java:3789) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$null$127(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.lambda$writeCollection$128(RpcConnection.java:3749) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1367) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1361) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.parse(RpcConnection.java:1355) ~[exist.jar:4.8.0]
	at sun.reflect.GeneratedMethodAccessor53.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
2022-08-23 10:26:39,524 [http-nio-8081-exec-3] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 10:26:43,978 [http-nio-8081-exec-1] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 10:27:54,815 [http-nio-8081-exec-7] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 10:30:19,316 [http-nio-8081-exec-8] INFO  (RpcConnection.java [doQuery]:257) - query took 2ms. 
2022-08-23 10:30:26,602 [http-nio-8081-exec-6] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 10:31:00,715 [http-nio-8081-exec-5] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-23 10:37:20,902 [http-nio-8081-exec-2] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-23 10:37:30,357 [http-nio-8081-exec-8] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 10:37:34,237 [http-nio-8081-exec-5] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 10:37:40,473 [http-nio-8081-exec-3] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 10:37:40,490 [http-nio-8081-exec-6] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-23 10:37:53,828 [http-nio-8081-exec-7] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 10:37:53,843 [http-nio-8081-exec-10] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 10:37:57,382 [http-nio-8081-exec-5] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-23 10:37:57,396 [http-nio-8081-exec-4] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 11:04:25,290 [http-nio-8081-exec-2] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-23 11:04:41,338 [http-nio-8081-exec-6] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 11:05:26,448 [http-nio-8081-exec-7] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 11:06:27,658 [http-nio-8081-exec-7] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 11:08:06,528 [http-nio-8081-exec-3] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 11:14:40,905 [http-nio-8081-exec-4] ERROR (XmlRpcErrorLogger.java [log]:36) - Failed to invoke method retrieveFirstChunk in class org.exist.xmlrpc.RpcConnection: result set unknown or timed out: 29 
org.apache.xmlrpc.common.XmlRpcInvocationException: Failed to invoke method retrieveFirstChunk in class org.exist.xmlrpc.RpcConnection: result set unknown or timed out: 29
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:129) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.execute(ReflectiveXmlRpcHandler.java:106) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServerWorker.execute(XmlRpcServerWorker.java:46) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServer.execute(XmlRpcServer.java:86) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcStreamServer.execute(XmlRpcStreamServer.java:200) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServletServer.execute(XmlRpcServletServer.java:112) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServlet.doPost(XmlRpcServlet.java:196) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.exist.xmlrpc.RpcServlet.doPost(RpcServlet.java:99) [exist.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:681) [servlet-api.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:462) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:356) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:316) [catalina.jar:8.5.73]
	at org.exist.http.urlrewrite.Forward.doRewrite(Forward.java:51) [exist-optional.jar:4.8.0]
	at org.exist.http.urlrewrite.XQueryURLRewrite.service(XQueryURLRewrite.java:206) [exist-optional.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52) [tomcat-websocket.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.logging.log4j.web.Log4jServletFilter.doFilter(Log4jServletFilter.java:71) [log4j-web-2.15.0.jar:2.15.0]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:196) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97) [catalina.jar:8.5.73]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:698) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78) [catalina.jar:8.5.73]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:364) [catalina.jar:8.5.73]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:624) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:831) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1673) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-util.jar:8.5.73]
	at java.lang.Thread.run(Thread.java:750) [?:1.8.0_342]
Caused by: org.exist.EXistException: result set unknown or timed out: 29
	at org.exist.xmlrpc.RpcConnection.lambda$retrieveFirstChunk$72(RpcConnection.java:2275) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.retrieveFirstChunk(RpcConnection.java:2272) ~[exist.jar:4.8.0]
	at sun.reflect.GeneratedMethodAccessor54.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
2022-08-23 11:14:48,117 [http-nio-8081-exec-5] ERROR (XmlRpcErrorLogger.java [log]:36) - Failed to invoke method retrieveFirstChunk in class org.exist.xmlrpc.RpcConnection: result set unknown or timed out: 29 
org.apache.xmlrpc.common.XmlRpcInvocationException: Failed to invoke method retrieveFirstChunk in class org.exist.xmlrpc.RpcConnection: result set unknown or timed out: 29
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:129) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.execute(ReflectiveXmlRpcHandler.java:106) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServerWorker.execute(XmlRpcServerWorker.java:46) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcServer.execute(XmlRpcServer.java:86) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.server.XmlRpcStreamServer.execute(XmlRpcStreamServer.java:200) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServletServer.execute(XmlRpcServletServer.java:112) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.apache.xmlrpc.webserver.XmlRpcServlet.doPost(XmlRpcServlet.java:196) [xmlrpc-server-3.1.3.jar:3.1.3]
	at org.exist.xmlrpc.RpcServlet.doPost(RpcServlet.java:99) [exist.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:681) [servlet-api.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:462) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:356) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:316) [catalina.jar:8.5.73]
	at org.exist.http.urlrewrite.Forward.doRewrite(Forward.java:51) [exist-optional.jar:4.8.0]
	at org.exist.http.urlrewrite.XQueryURLRewrite.service(XQueryURLRewrite.java:206) [exist-optional.jar:4.8.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) [servlet-api.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52) [tomcat-websocket.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.logging.log4j.web.Log4jServletFilter.doFilter(Log4jServletFilter.java:71) [log4j-web-2.15.0.jar:2.15.0]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.73]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:196) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97) [catalina.jar:8.5.73]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81) [catalina.jar:8.5.73]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:698) [catalina.jar:8.5.73]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78) [catalina.jar:8.5.73]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:364) [catalina.jar:8.5.73]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:624) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65) [tomcat-coyote.jar:8.5.73]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:831) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1673) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-coyote.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659) [tomcat-util.jar:8.5.73]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-util.jar:8.5.73]
	at java.lang.Thread.run(Thread.java:750) [?:1.8.0_342]
Caused by: org.exist.EXistException: result set unknown or timed out: 29
	at org.exist.xmlrpc.RpcConnection.lambda$retrieveFirstChunk$72(RpcConnection.java:2275) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.function.XmlRpcFunction.apply(XmlRpcFunction.java:45) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3958) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.withDb(RpcConnection.java:3942) ~[exist.jar:4.8.0]
	at org.exist.xmlrpc.RpcConnection.retrieveFirstChunk(RpcConnection.java:2272) ~[exist.jar:4.8.0]
	at sun.reflect.GeneratedMethodAccessor54.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_342]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_342]
	at org.apache.xmlrpc.server.ReflectiveXmlRpcHandler.invoke(ReflectiveXmlRpcHandler.java:115) ~[xmlrpc-server-3.1.3.jar:3.1.3]
	... 43 more
2022-08-23 11:14:55,867 [http-nio-8081-exec-6] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 11:21:02,581 [http-nio-8081-exec-2] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 11:22:59,828 [http-nio-8081-exec-1] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 11:26:34,478 [http-nio-8081-exec-3] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 12:50:32,213 [http-nio-8081-exec-2] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 12:50:57,736 [http-nio-8081-exec-8] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 12:50:57,858 [http-nio-8081-exec-1] INFO  (RpcConnection.java [createCollection]:184) - collection /db/certificates has been created 
2022-08-23 12:52:08,562 [http-nio-8081-exec-1] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 12:52:21,859 [http-nio-8081-exec-1] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 12:55:46,035 [http-nio-8081-exec-5] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 12:55:52,865 [http-nio-8081-exec-9] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 12:55:52,882 [http-nio-8081-exec-1] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-23 12:55:57,200 [http-nio-8081-exec-4] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 12:55:57,215 [http-nio-8081-exec-2] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-23 12:55:57,666 [http-nio-8081-exec-5] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 12:55:57,686 [http-nio-8081-exec-6] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 12:56:05,122 [http-nio-8081-exec-8] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 12:56:05,137 [http-nio-8081-exec-7] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-23 12:57:11,746 [http-nio-8081-exec-2] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 12:57:11,763 [http-nio-8081-exec-4] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 13:31:38,979 [http-nio-8081-exec-1] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 13:31:41,730 [http-nio-8081-exec-4] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 13:31:41,749 [http-nio-8081-exec-2] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 13:43:23,550 [http-nio-8081-exec-8] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 13:43:23,581 [http-nio-8081-exec-9] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 13:44:52,054 [http-nio-8081-exec-1] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 13:44:55,604 [http-nio-8081-exec-4] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 13:44:55,625 [http-nio-8081-exec-2] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 14:01:14,570 [http-nio-8081-exec-7] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 14:01:14,597 [http-nio-8081-exec-8] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 14:01:42,191 [http-nio-8081-exec-5] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 14:01:42,206 [http-nio-8081-exec-6] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 14:01:42,965 [http-nio-8081-exec-6] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-23 14:01:42,980 [http-nio-8081-exec-1] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 14:01:44,306 [http-nio-8081-exec-4] INFO  (RpcConnection.java [doQuery]:257) - query took 1ms. 
2022-08-23 14:01:44,324 [http-nio-8081-exec-2] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 14:01:46,192 [http-nio-8081-exec-5] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 14:01:46,209 [http-nio-8081-exec-6] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 14:01:47,363 [http-nio-8081-exec-8] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 14:01:47,377 [http-nio-8081-exec-7] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 14:01:47,455 [http-nio-8081-exec-10] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 14:01:47,477 [http-nio-8081-exec-3] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 14:01:47,648 [http-nio-8081-exec-9] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
2022-08-23 14:01:47,666 [http-nio-8081-exec-8] INFO  (RpcConnection.java [doQuery]:257) - query took 0ms. 
